bot.py

```diff
@@ -41,6 +41,7 @@
     "**CML show IP addresses** > show IP addresses\n",
     "**CML show server utilization** > show current CPU and Memory usage\n",
     "**CML stop lab** > stop labs of your choice\n",
+    "**Request static IP address** > allocates an static ip address for CML\n",
```

Comment:
Change to "allocates static ip addresses for CML"

Current wording makes it seem singular, and from what I remember you can request up to 10 at a time

Reply:
it is singular - can request up to 10 but 1 at a time